_includes/search.html

```diff
@@ -1,4 +1,4 @@
-<div class="row">
+<div class="row" id="filter-section">
```

Comment:
Is this change related to this PR?

Reply:
I included it when I was trying to hide sections before I clarified during the standup. I'll take it out now